src/components/Guides/PlaylistCard.tsx

```diff
@@ -19,20 +21,31 @@ const PlaylistCard = (props: PlaylistCardProps) => {
   const lesson = props.lesson;
 
   const resources = useSelector((state: RootState) => state.resources);
+  const topics = useSelector((state: RootState) => state.topics);
   const countMedia = (media: string) =>
     lesson.resourceIDs.filter((id) => resources[id].type == media).length;
 
   const numVideos = countMedia('Video');
   const numArticles = countMedia('Article');
+  const mySet = new Set<Topic>();
+  lesson.resourceIDs.map((resID) =>
+    resources[resID].tags.map((tag) => mySet.add(topics[tag]))
+  );
 
   return (
     <Card className={classes.card}>
       <h2 className={classes.title}>{lesson.title}</h2>
       <div className={classes.spread}>
         <div className={classes.column}>
-          {lesson.objectives.map((tag) => (
-            <Tag tag={tag} />
+          {Array.from<Topic>(mySet).map((topic: Topic) => (
```

Comment:
Oo also one design note, we'd like to limit the number of tags to 3 (so only map the first three elements out of all the topics)!